installing/installing_vsphere/installing-restricted-networks-vsphere.adoc

```diff
@@ -41,7 +41,12 @@ include::modules/installation-about-restricted-network.adoc[leveloffset=+1]
 
 include::modules/cluster-entitlements.adoc[leveloffset=+1]
 
-include::modules/installation-vsphere-infrastructure.adoc[leveloffset=+1]
+include::modules/
```

Comment:
```suggestion
```

Reply:
Slippy fingers must have caused this. Apologies. Fixed by readding the include text.